### DISCUSSION TITLE
VSCode extension should have separate commands for "enable" and "disable" copilot

### DISCUSSION THREAD

**Discussion by birgersp** [community/community#70311] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I find myself having enable/disable copilot multiple times throughout the day. Some times I need it, sometimes I need it to go away.

Currently, there is a single command to "enable/disable" copilot. So my flow is that I hit Ctrl+Shift+P then write copilot and get this:

![image](https://github.com/community/community/assets/5260237/2a837da9-6d82-4120-acd3-314014a79708)

I arrow down to it and hit Enter. Now I get a prompt at the bottom of the screen asking me if I really want to disable/enable copilot. I can't confirm the prompt with the keyboard, I actually have to move my mouse to the prompt to confirm. Mouse interaction is for dinosaurs.

Suggestion: split the single enable/disable command into two separate enable and disable commands. Don't prompt me if I really meant to trigger the command (obviously I did).

**Comment by Akash1134** [comment]
Hi @birgersp ,

Thank you for your valuable feedback. Have you tried creating custom keybindings for the enable and disable commands, and assigning them to keyboard shortcuts of your choice. 

- Open the Keyboard Shortcuts editor in Visual Studio Code and search for the "GitHub Copilot: Enable GitHub Copilot" command and click on the "+" icon next to it to add a keybinding.
- Press the keyboard shortcut that you want to use for enabling GitHub Copilot. 
- Now, as you desire - search for the "GitHub Copilot: Disable GitHub Copilot" command and click on the "+" icon next to it to add a keybinding.
- Press the keyboard shortcut that you want to use for disabling GitHub Copilot. For example, you might use `Ctrl+Alt+D` (Windows/Linux) or `Cmd+Option+D` (macOS).
- Save your changes and close the Keyboard Shortcuts editor.

**Reply by birgersp** [reply]
Thanks for you reply!

Unforunately I don't see the command you mention in your first bulletin.

This is what I get when searching for "GitHub Copilot: Enable GitHub Copilot"...

![image](https://github.com/community/community/assets/5260237/0096618a-bfcf-49bd-8079-d6f116cc005c)

**Comment by birgersp** [comment]
I guess I updated, and now the copilot extension no longer shows me a prompt when I run the "enable/disable" command. Neat!
